### DISCUSSION TITLE
Running fail2ban outside of docker-mailserver, directly on the host machine

### DISCUSSION THREAD

**Discussion by daks** [docker-mailserver/docker-mailserver#4250] [Q&A] [answered]
Hello,
I'm prototyping the use of docker-mailserver on Docker rootless with a dedicated user.
I read the [fail2ban doc page](https://docker-mailserver.github.io/docker-mailserver/edge/config/security/fail2ban/) and, if I'm not mistaken, it runs fail2ban inside the docker-mailserver container.

As I'm already running fail2ban on my host machine, outside of Docker, I thought I can continue to do it and just activate `postfix` and `dovecot` filters configuring them to point to the `docker-mailserver` logs.
Any opinion on this setup? Any potential problem?

For information my setup _(not validated at the moment because external ports are not accessible)_
```
# fail2ban.local on the host machine
[postfix]
enabled = true
backend = auto
mode = more
postfix_log = /path/to/mail-logs/mail.log

[dovecot]
enabled = true
backend = auto
dovecot_log = /path/to/mail.log

**Comment by casperklein** [comment, marked as answer]
> if I'm not mistaken, it runs fail2ban inside the docker-mailserver container.

Correct.

>I thought I can continue to do it and just activate postfix and dovecot filters configuring them to point to the docker-mailserver logs.

Yes, that should work.
